docs/instance-filtering-guide-en.md

```
@@ -0,0 +1,760 @@
+**Good:**
+```json
+{
+  "and": [
+    {"status": {"eq": "Active"}},
+    {"attributes": {"amount": {"gt": "500"}}}
+  ]
+}
+```
+
+**Also Works (Legacy):**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)